vital/bindings/python/vital/types/image_container.cxx

```diff
@@ -151,33 +132,23 @@ PYBIND11_MODULE(image_container, m)
         >>> assert np.all(np_img == np_img2)
     )")
 
-  .def(py::init(&new_cont), py::arg("image"))
-
-  .def_static("fromarray",
-    [](py::array& arr) -> s_image_cont {
-      auto locals = py::dict(py::arg("arr")=arr);
-      py::exec(R"(
-          from vital.types import ImageContainer
-          from PIL import Image as PILImage
-          from vital.util import VitalPIL
-          pil_img = PILImage.fromarray(arr)
-          vital_img = VitalPIL.from_pil(pil_img)
-          self = ImageContainer(vital_img)
-          )", py::globals(), locals);
-      return locals["self"].cast<s_image_cont>();
-    }, py::doc(R"(
-    Create an ImageContainer from a numpy array
-
-    Example:
-        >>> from vital.types import ImageContainer
```

Comment:
Are these tests replicated somewhere else?

Reply:
Not as a doc test. However, [fromarray test in image container](https://github.com/Kitware/kwiver/pull/818/files/8d3daa35c8d298c057a4c239e191cf6585a43e90#diff-3a76da4c16d20f43318ff80409d1380dR80) checks for it with all types we support.